examples/more/components/library/phrases-config.cmake.in

```diff
@@ -1,3 +1,16 @@
+@PACKAGE_INIT@
+
+# valid components that can be searched for
+set(_phrases_supported_components greetings farewells)
+
+# iterate through components attempting to be found
 foreach(component ${@PROJECT_NAME@_FIND_COMPONENTS})
```

Comment:
I think using `${@PROJECT_NAME@_FIND_COMPONENTS}` here is incorrect. That assumes that the call to `find_package()` uses `@PROJECT_NAME@` as the name of the package, but that precludes things like `find_package(someOtherName NAMES realName)`. You probably should be using `${${CMAKE_FIND_PACKAGE_NAME}_FIND_COMPONENTS}` instead.

Reply:
Thanks for taking a look at the PR @craigscott-crascit! Much appreciated 🙂

Good spot! I'll get this updated, tested and merged soon, I'll tag you on the PR when I publish it for visibility.

Cheers!

Reply:
Updated as part of commit 61dc781fadf3b3e758c9d6f422377a6ba5d34088, thanks!